Sources/Machine/rbm_spin.cc

```diff
@@ -46,20 +63,23 @@ RbmSpin::RbmSpin(std::shared_ptr<const AbstractHilbert> hilbert, Index nhidden,
                      << nhidden << " != " << alpha << " * " << nvisible);
   }
   nhidden = std::max(nhidden, alpha * nvisible);
-
-  W_.resize(nvisible, nhidden);
+  W_.resize(nvisible, nhidden); 
   if (usea) {
     a_.emplace(nvisible);
   }
   if (useb) {
     b_.emplace(nhidden);
   }
-
+        
```

Comment:
did you use `clang-format` before committing? check the dev docs in the repo directory

Reply:
Yes, I checked the formatting with clang.